lib/tarearbol/application.ex

```diff
@@ -23,6 +23,15 @@ defmodule Tarearbol.Application do
     Task.Supervisor.children(Tarearbol.Application)
   end
 
+  def jobs do
+    Tarearbol.Application.children
+    |> Enum.map(&Process.info/1)
```

Comment:
From Elixir docs (https://hexdocs.pm/elixir/Process.html#info/1):
> Use this only for debugging information.

But info/2 hasn't the same annotation (and is probably lighter because it does not get all the debug info from the process, including full call stack, garbage collection status, message_queue_data, etc ?)
```ex
Tarearbol.Application.children
|> Enum.map(&Process.info/2(:dictionary))
|> Enum.map(fn kw -> kw[:job] end)
|> Enum.filter(& &1)